### ISSUE TITLE
Don't sync pos/rot/scale via .setAttribute

### ISSUE THREAD

**Issue by ngokevin** [networked-aframe/networked-aframe#106]
Use `el.object3D.position.set(x, y, z)` or whatever versus `el.setAttribute('position', data)`. 

The former has just about zero overhead. The latter has quite a bit. The data will still be synced on getAttribute. Save a lot of time on ticks.

**Comment by haydenjameslee** [comment]
I think the only thing this would break is the existing `lerp` component, since it listens to an `oncomponentchanged` event which wouldn't get fired in this optimization. That being said however, we're looking into changing how that lerp component works to make it more explicit. We could make both changes at the same time.

**Comment by ngokevin** [comment]
I wouldn't use `componentchanged` for anything, particularly for rapidly changing stuff. pos/rot will always be changing, I'd run a continuous tick. Yeah, lerp should stop using setAttribute (and most ticks for that matter) and go for the object3D API.

**Comment by haydenjameslee** [comment]
Oh i was wrong, turns out the lerp doesn't use the component changed event anymore: https://github.com/haydenjameslee/aframe-lerp-component/blob/master/index.js#L41

**Comment by ngokevin** [comment]
Right, I think we've talked about that before haha. That could have some configurable check to not do the `checkForComponentChanged` in some cases...like head pose or controller pose is always going to change.

Okay, so on the NAF side, this isn't a problem, because it always uses lerp for pos/rot?

**Comment by haydenjameslee** [comment]
Its still a problem, it basically changes the position _twice_ because of this. Once via .setAttribute and once via object3D. NAF uses .setAttribute then the lerp comp uses object3D. Instead we are going to remote that lerp component and write our own NAF lerper that uses object3D directly.

**Comment by haydenjameslee** [comment]
@netpro2k do you have a prototype of this new lerper in action already?

**Comment by netpro2k** [comment]
Nothing beyond what we discussed in the past on slack, still not sure what the best path forward with that is.

**Comment by haydenjameslee** [comment]
+1

**Comment by vincentfretin** [comment]
In current code, when lerping is used, tick uses `object3D.position.copy(position)`, so I'm closing this issue.
